aries-site/src/examples/templates/filtering/FilteringLists/FilteringLists.js

```diff
@@ -88,17 +96,24 @@ export const FilteringLists = ({ containerRef }) => {
           Orders
         </Heading>
       </Header>
-      <FiltersProvider>
-        <Box gap="medium">
-          <FilterControls
-            data={orders}
-            filters={filtersConfig}
-            layerProps={layerProps}
-            searchFilter={{ placeholder: 'Search' }}
-          />
+      <Box fill>
+        <Data
+          data={orders}
+          updateOn="submit"
+          height={{ min: 'medium', max: '100%' }}
+        >
+          <Toolbar>
+            <DataSearch />
+            <DataFilters layer={layerProps}>
```

Comment:
I think we should have this as a side drawer layer instead of center

Reply:
The existing example on the Design Site [here](https://design-system.hpe.design/templates/filtering#filters-in-a-center-layer) has a layer that is centered and has a horizontal direction, so I left the `layerProps` here. Will connect with Eric about direction on this.

Reply:
I'm inclined to go with Taylor's inclination here.